### PR TITLE
fix copyright headers

### PR DESCRIPTION
**Pull request by ishmum123**

### CODE REVIEW

File: dumper/app/build.gradle [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022 Google LLC
+ * Copyright 2023 Google LLC
```
Comment (dawidxc) [inline dumper/app/build.gradle:2]: Should it be `2023` or `2022-2023`?

Comment (ishmum123) [reply dumper/app/build.gradle:2]: Let's wait for the CW team's answer.

cc: @shevek-google @paolomorandini

File: gradle/codequality/HEADER.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022 Google LLC
+ * Copyright 2023 Google LLC
```
Comment (paolomorandini) [inline gradle/codequality/HEADER.java:2]: Should it be `2023` or `2022-2023`?

Comment (shevek-google) [inline gradle/codequality/HEADER.java:2]: Should this be 2022-2023?

**Review by shevek-google — CHANGES_REQUESTED** [review]

File: gradle/codequality/HEADER.java (hunk 2 above)
Comment (ishmum123) [reply gradle/codequality/HEADER.java:2]: We have been discussing this [here](https://github.com/google/dwh-migration-tools/pull/159#discussion_r1136933548). What's your thoughts?

Comment (ishmum123) [reply gradle/codequality/HEADER.java:2]: Changing to 2022-2023...

**Review by dawidxc — APPROVED** [review]

**Review by shevek-google — APPROVED** [review]

**Pull request merged**